lib/java/com/google/android/material/floatingactionbutton/FloatingActionButton.java

```diff
@@ -246,6 +246,8 @@ public FloatingActionButton(
     boolean ensureMinTouchTargetSize = a
         .getBoolean(R.styleable.FloatingActionButton_ensureMinTouchTargetSize, false);
 
+    setEnabled(a.getBoolean(R.styleable.FloatingActionButton_android_enabled, true));
```

Comment:
Is this required? I think the super class would handle setting the enabled state based on the `android:enabled` attribute.

Reply:
It doesn't seem to. Tracing it back: `FloatingActionButton` <- `VisibilityAwareImageButton` <- `ImageButton` <- `ImageView` <- `View` - the source suggests none of these classes parse/handle the `android:enabled` attribute

Reply:
Yep that's right!